volume/devicemapper/devmapper/deviceset.go

```diff
@@ -1127,7 +1127,7 @@ func (devices *DeviceSet) checkGrowBaseDeviceFS(info *devInfo) error {
 	}
 
 	if devices.baseFsSize < devices.getBaseDeviceSize() {
-		return fmt.Errorf("devmapper: Base device size cannot be smaller than %s", units.HumanSize(float64(devices.getBaseDeviceSize())))
+		return fmt.Errorf("devmapper: Base device size cannot be smaller than %s", units.BytesSize(float64(devices.getBaseDeviceSize())))
```

Comment:
Will the string output here match the spelling of the viable config option parsed by Docker?